zebedee-cms/src/main/java/com/github/onsdigital/zebedee/util/EncryptionUtils.java

```diff
@@ -64,4 +68,16 @@ public static InputStream encryptionInputStream(InputStream inputStream, SecretK
     public static InputStream encryptionInputStream(Path path, SecretKey key) throws IOException {
         return encryptionInputStream(Files.newInputStream(path), key);
     }
+
+    public static String createMD5HashSum(String value) {
```

Comment:
I think `Checksum` is the term that's normally used, rather than `HashSum`.

Reply:
have addressed this now